project-keeper/src/main/java/com/exasol/projectkeeper/validators/pom/PomFileValidator.java

```diff
@@ -14,14 +14,14 @@
 import com.exasol.projectkeeper.Validator;
 import com.exasol.projectkeeper.config.ProjectKeeperConfig;
 import com.exasol.projectkeeper.validators.files.RequiredFileValidator;
-import com.exasol.projectkeeper.validators.finding.SimpleValidationFinding;
-import com.exasol.projectkeeper.validators.finding.ValidationFinding;
+import com.exasol.projectkeeper.validators.finding.*;
```

Comment:
not sure about `Java` but in `Python` I generally would discourage `*` imports

Reply:
That's automated by our formatter. After a certain amount of imports it starts grouping.  @redcatbear created a config according to our style.